cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -26,6 +46,9 @@ class GameWorldService {
         post_world_initialization_handle_ = FWorldDelegates::OnPostWorldInitialization.AddRaw(this, &GameWorldService::postWorldInitializationHandler);
         world_cleanup_handle_ = FWorldDelegates::OnWorldCleanup.AddRaw(this, &GameWorldService::worldCleanupHandler);
 
+        using TMapStringToString = std::map<std::string, std::string>;
+        using TMapStringToInt = std::map<std::string, uint64_t>;
+
```

Comment:
Let's remove `open_level` and `set_game_paused` as entry points because the underlying Unreal functions are `UFUNCTIONS` that can already be called using `call_function`.

In future, I would maybe be willing to expose `set_game_paused` directly, because it will be called every frame in most applications, and therefore it might deserve its own native code path that doesn't go through `Unreal::callFunction(...)`.

But we need a good reason for exposing some native entry points and not others, because it will lead to an unlimited amount of code clutter if we exposed every `UFUNCTION` with its own entry point. And if we decide to do this, we should place these native entry points in their own service (e.g., `GameWorldServiceNative`) because `GameWorldService` is complicated enough already. And we should wait to see some kind of experimental result that shows a measurable performance improvement using the native variant before we decide to clutter our code.

Reply:
I agree with your suggestion. I'll make these changes in the next PR, where I'll be exposing functions from `UnrealClassRegistrar` as it'd be more consistent that way to have all functions of `UnrealClassRegistrar` exposed in the same PR.

Reply:
Cool. I like that idea. In that case, `open_level` and `set_game_paused` can stay.